src/coreclr/vm/ClrEtwAll.man

```diff
@@ -3652,6 +3685,12 @@
                            task="Contention"
                            symbol="ContentionStop_V1" message="$(string.RuntimePublisher.ContentionStop_V1EventMessage)"/>
 
+                    <event value="414" version="0" level="win:Informational"  template="AwareLockCreated"
```

Comment:
The name `AwareLock` is specific to the current implementation. The implementation may be changed, or another implementation may be added in the future (eg. https://github.com/dotnet/runtime/issues/34812) and may want to reuse the same events, maybe just `LockCreated` would be more generic

Reply:
Agreed, this is a good idea.